packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/use-flow.md

```diff
@@ -1,35 +1,38 @@
-The following sections outline the main requests required to implement the Authorization Code flow using direct calls to Okta's [OIDC & OAuth 2.0 API](/docs/reference/api/oidc/). Typically, you don't need to make direct calls to the [OIDC & OAuth 2.0 API](/docs/reference/api/oidc/) if you're using one of Okta's SDKs.
+The following sections outline the main requests required to implement the Authorization Code flow using direct calls to the [OIDC & OAuth 2.0 API](/docs/reference/api/oidc/). Typically, you don't need to make direct calls if you're using one of the Okta SDKs.
 
 ### Request an authorization code
 
-To get an authorization code, your app redirects the user to your [authorization server's](/docs/concepts/auth-servers/) `/authorize` endpoint. If you are using the org authorization server, then your request URL would look something like this:
+To get an authorization code, your app redirects the user to your [authorization server's](/docs/concepts/auth-servers/) `/authorize` endpoint. If you're using the org authorization server, then your request URL would look something like this:
 
-```bash
-https://${yourOktaDomain}/oauth2/v1/authorize?client_id=0oabucvy
-c38HLL1ef0h7&response_type=code&scope=openid&redirect_uri=https%3A%2F%2Fexample.com&state=state-296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
+```uri
```

Comment:
```suggestion
```bash
```
I don't think VuePress supports `uri` as a code block language? Use either `bash` or `http`

Reply:
Weirdly, prismjs does support uri as a language code, but it turns out it's only there so URLs can be embedded in other language codes like http or bash. (https://github.com/PrismJS/prism/issues/3439) Strikes me they shouldn't make it public that it's a thing they do without more effort on your own part.

Net result: I'll change it back to bash.